ocfweb/docs/docs/staff/procedures/gapps.md

```diff
@@ -0,0 +1,27 @@
+[[!meta title="Making OCF Google Apps accounts"]]
+
+The OCF has its own Google Apps deployment, which we mainly use for GDrive and
+GMail. While it's possible to use Calnet accounts for these (ocf.berkeley.edu
+emails redirect to Calnet by default), most active staffers choose to have a
+dedicated account because:
+
+* It allows for nicer integration with our Google Drive folders
+* It makes it easier to send emails from the ocf.berkeley.edu domain, which is
+good if you want to send an email while representing the OCF
+* It can be useful to have a separation between OCF and Calnet accounts.
+
+## Making an account
+Google Apps accounts are available upon request for **any OCF staffer**. Before
```

Comment:
I'm 2/10 on this policy. The existing "increased privileges" policy of "be staff for at least a semester, need it, or otherwise until people recognize you" with SM discretion otherwise is fine and we should stick to it. A lot of people drop by for one semester and then never show up again and the existing policy is appropriately balanced to keep people from farming us for GApps accounts while still being flexible enough to give people all the privileges they need as they need them.

At the very least, this is a policy change that should probably be reviewed by BoD, though as an SM, you do have the privilege to [amend](https://www.ocf.berkeley.edu/docs/staff/policies/staff-policy/#h2_amending-of-certain-policies) some of these policies yourself.

Reply:
I'd be happy to have a BoD discussion about this, though I don't really see what the issue is here. GApps accounts don't actually give extra privileges, and "GApps account farming" isn't actually a serious issue for us.